lib/framework.js

```diff
@@ -302,7 +302,8 @@ export default class Framework {
 		if (this.config.ui5.type) {
 			return;
 		}
-		const filePath = path.join(this.config.basePath, "ui5.yaml");
+		const filePath = this.config.ui5.configPath ?
+			path.resolve(this.config.ui5.configPath) : path.join(this.config.basePath, "ui5.yaml");
```

Comment:
This should be aligned with the existing logic:
https://github.com/SAP/karma-ui5/blob/c5ddfc5b94eba5551dac3c79752b56a3455c7264/lib/framework.js#L472

The `ui5.configPath` is resolved relative to `basePath`.

Reply:
Thanks!